src/loaders/workerTypes.js

```diff
@@ -49,11 +50,41 @@ export default ({ queue, awsProvisioner }) => {
       })
     )
   );
+  const awsProvisionerRecentErrors = new DataLoader(queries =>
+    Promise.all(
+      queries.map(async ({ filter }) => {
+        const recentErrors = (await ec2Manager.getRecentErrors()).errors;
+        const { region, az, instanceType, ...errors } = filter
+          ? sift(filter, recentErrors)
+          : recentErrors;
+
+        return new Aws(region, az, instanceType, errors);
+      })
+    )
+  );
+  const awsProvisionerHealth = new DataLoader(queries =>
+    Promise.all(
+      queries.map(async ({ filter }) => {
+        const data = await ec2Manager.getHealth();
+        const health = filter ? sift(filter, data) : data;
+        const createEntity = ({ region, az, instanceType, ...health }) =>
+          new Aws(region, az, instanceType, health);
+
+        return {
+          running: health.running.map(createEntity),
```

Comment:
Yeah, since you aren't doing the underscore mapping in the entity, this no longer has value. You should be able to pass these directly now:

```js
const health = await ec2Manager.getHealth();

return filter ? sift(filter, health) : health;
```